### ISSUE TITLE
Something is not right. 

### ISSUE THREAD

**Issue by PranavBhattarai** [Magisk-Modules-Repo/Detach#4]
The entire option of kill, add app in Black list, help, etc should be inside "su Detach" or "Detach" option like many normal module out there (see second & third screenshot). Right now, its very disturbing place.

![Screenshot_20190413-074318](https://user-images.githubusercontent.com/32763112/56073209-db274080-5d8f-11e9-9da9-196c58d74635.jpg)

The more comfortable Standard sexy way:
![Screenshot_20190413-075315](https://user-images.githubusercontent.com/32763112/56073308-1aa25c80-5d91-11e9-83e5-b2ff84db9369.jpg)

![Screenshot_20190413-075339](https://user-images.githubusercontent.com/32763112/56073313-30b01d00-5d91-11e9-85a7-4fb1b2fb57dc.jpg)

I wish, Detaching was more look alike like Debloat module. (Debloat uninstall apps systemlessly) (Detach remove apps from Play store systemlessly)

I think u should collaborate with other module creator, to solve this detaching problem. It's just don't work properly and sometimes it doesn't work at all. Reboot is not sustain. 

(For now, I would be leaving/uninstalling this module. It's hopeless for now. 😑 But it's a nice approach/concept. ♥️)

**Comment by xerta555** [comment]
@PranavBhattrai Hello, can you try again please ?
I just push an update.

**Comment by PranavBhattarai** [comment]
@xerta555  Well, it is working for now. It sustained the reboot too. 

(Hope, it remain stable. Usually, it doesn't. I use this for only 1 app, i.e YouTube. So that I can install YouTubeVanced from Magisk module which is biggest lifehack app. 

And one thing is my phone is scheduled to shut down from 12 A.M to 4 A.M every single day at night while I'm asleep. 

That's is when your module should work as it without any issue. ) 

For now, it seems good, but who knows. I am gonna keep it. I really need it.

(Feature Request: The terminal way of doing things should be more like "debloat" module, it would make the user's life easy. Right now, it's not that fast. I have to type in Capital "Detach" and use "help" section to find command. That's not easy subtle to do things. Please do install "Debloat" and notice the terminal's way of doing things, and see how comfortable it is. )

**Comment by xerta555** [comment]
> (Feature Request: The terminal way of doing things should be more like "debloat" module, it would make the user's life easy. Right now, it's not that fast. I have to type in Capital "Detach" and use "help" section to find command. That's not easy subtle to do things. Please do install "Debloat" and notice the terminal's way of doing things, and see how comfortable it is. )

When the terminal-menu was first added, because other options became necessary, it was similar to the style-module you mentioned.

The ergonomic style change came after several requests from users who wanted to be able to automate the detachment of applications from Play Store updates.

Honestly I don't see how to simplify the module any further since I have redesigned the help page several times with an upstream test between each modification.

The only change I can think of would be the creation of a symbolic link from the menu file to a shortcut file placed in the default Termux home folder, to name a few. This would allow the addition of a text help to indicate to the user that commands must be preceded by the root access command or directly typed in root mode.
Only for this method to work, the /data partition of the device must be in EXT format filesystem, which is not necessarily the case on all Android devices, which is why I didn't add this additional help feature.

I just added a small portion of explanatory text for the possible commands in the terminal without going into detail https://github.com/Magisk-Modules-Repo/Detach/commit/5e8b97fc7eaee68ddcccb855a6fb61e1b2469681.

Concerning your log file suggestion: https://github.com/Magisk-Modules-Repo/Detach/issues/2, the users of the module have never had any problems explaining their problems to me, they provided me with screenshots when necessary, but nothing very serious.
Honestly, I don't think that adding a log file listing everything that happens in the terminal when using the module is so important given the few critical actions that the module performs.
